libr/asm/asm.c

```diff
@@ -195,6 +195,11 @@ R_API RAsm *r_asm_new() {
 	if (!a) {
 		return NULL;
 	}
+	a->num = r_num_new (NULL, NULL, NULL);
```

Comment:
i dont think RAsm should creat that instance here, mainly because it is leaking because asm->num is set from core and rasm already.

Reply:
ok 810ea16d67cd4b26f319b3d36092a7c6ca1d3b16 ... also removed refcounting from RNum since it's clearly not needed now cc9f9433733ce25bf5ba761bedd8807d6a35a3dd